usaspending_api/api_contracts/contracts/v2/federal_accounts/federal_account_id/available_program_activity.md

```diff
@@ -0,0 +1,68 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# List of Available Program Activities [/api/v2/federal_accounts/{federal_account_id}/available_program_activity]
```

Comment:
Should we make this plural (`available_program_activities`) for consistency with `available_object_classes`?